common/djangoapps/student/api.py

```diff
@@ -92,13 +97,7 @@ def create_manual_enrollment_audit(
     else:
         enrollment = None
 
-    _create_manual_enrollment_audit(
```

Comment:
I just let my IDE reformat  according to linting rules; since it was only a couple of minor changes I didn't prevent it from doing that.